src/cloud-code/httpRequest.js

```diff
@@ -1,25 +1,36 @@
 var request = require("request"),
   Parse = require('parse/node').Parse;
 
-var encodeBody = function(body, headers = {}) {
+var encodeBody = function(options = {}) {
```

Comment:
We should use destructuring here instead of doing `headers || {}` - provides a slightly more readable public interface as well....